### ISSUE TITLE
[FR]: status "has new articles" vs "no errors" ?

### ISSUE THREAD

**Issue by Owyn** [martinrotter/rssguard#631]
### Brief description of the feature request

I have two feeds, when new articles arrive - one feeds status changes to "has new articles" and its color turns blue, while the others stats becomes "no errors" and color doesn't change but text becomes bold

![изображение](https://user-images.githubusercontent.com/1309656/153611211-41b29252-0d55-4a39-9be5-8cb807c9e28a.png)
![изображение](https://user-images.githubusercontent.com/1309656/153611269-6569e995-189d-4eb3-84e6-b76cd5d1161c.png)

so it'd be nice if instead of becoming blue and bold feed with new articles would just become bold and my system colors theme could be preserved that way they way it is preserved with json feeds of "no error" status when new articles arrive there

**Comment by martinrotter** [comment]
This is classic RSS/ATOM/JSON plugin? Please post URLs of both feeds.

When item (article list or feed list) is BOLD -> it contains some unread (and not necessarily new) articles.
When item in feed list is BLUE -> it contains totally new articles (which were not present in feed previously) and also these new articles do not need to be necessarily unread (this happens particularly for synced plugins like Feedly etc.).

It may happen that feed has unread articles but no new articles etc.

**Comment by martinrotter** [comment]
Yes. You are correct, "new messages" status is overwritten each time feed is synced. So if during first sync you have new messages feed turns blue, but in the second sync no new messages are there and feed turns black/bold again.

Let me check.
